frontend/src/app/users/personal-access-tokens/personal-access-tokens.component.html

```diff
@@ -299,20 +299,27 @@ <h2 class="mb-2 text-xl font-medium">Token Overview</h2>
               }
             </mat-expansion-panel>
 
-            <button
-              class="mt-2 w-full"
-              (click)="deleteToken(token)"
-              mat-raised-button
+            <div
+              class="mt-2"
+              matTooltip="Managed tokens can't be deleted manually."
+              [matTooltipDisabled]="!token.managed"
             >
-              <mat-icon color="warn">delete</mat-icon>
-              <span>
-                @if (isTokenExpired(token.expiration_date)) {
-                  Remove token from overview
-                } @else {
-                  Revoke token
-                }
-              </span>
-            </button>
+              <button
+                class="w-full"
+                (click)="deleteToken(token)"
+                mat-raised-button
+                [disabled]="token.managed"
+              >
+                <mat-icon color="warn">delete</mat-icon>
+                <span>
+                  @if (isTokenExpired(token.expiration_date)) {
+                    Remove token from overview
+                  } @else {
+                    Revoke token
+                  }
+                </span>
+              </button>
+            </div>
```

Comment:
Admins can revoke managed tokens. Do we only want to make this possible via the API, or should this UI allow it if viewed by an admin?

Reply:
This should only be possible via the new global token overview, not in the personal user profile: https://github.com/DSD-DBS/capella-collab-manager/issues/2152